src/components/Card.tsx

```diff
@@ -0,0 +1,20 @@
+import OpenColor from "open-color";
+
+import "./Card.scss";
+
+export const Card: React.FC<{
+  color: keyof OpenColor;
+}> = ({ children, color }) => {
+  return (
+    <div
+      className="Card"
```

Comment:
`excalidraw-card` ?

Reply:
Forgot about this one. But namespacing it should be enough as this component lives inside the container (we don't prefix our other components either).